src/test/scala/uk/gov/nationalarchives/tdr/api/utils/FixedUUIDMultipleSource.scala

```diff
@@ -0,0 +1,18 @@
+package uk.gov.nationalarchives.tdr.api.utils
+
+import java.util.UUID
+
+import uk.gov.nationalarchives.tdr.api.service.UUIDSource
+
+object FixedUUIDMultipleSource extends UUIDSource {
+  var idx = -1
+  override def uuid: UUID = {
+    idx += 1
+    List(
+    UUID.fromString("bbcae38a-4334-41a5-9564-99ad90b7254c"),
+    UUID.fromString("bbcae38a-4334-41a5-9564-99ad90b7254c"),
+    UUID.fromString("bbcae38a-4334-41a5-9564-99ad90b7254c")
```

Comment:
Are these supposed to be different UUIDs? Or has this class been replaced by the next one?

Reply:
Good spot, this is unused now, I've deleted it.